applications/matter_bridge/src/bridged_device_types/temperature_sensor.cpp

```diff
@@ -98,54 +98,65 @@ CHIP_ERROR TemperatureSensorDevice::HandleReadTemperatureMeasurement(AttributeId
 CHIP_ERROR TemperatureSensorDevice::HandleAttributeChange(chip::ClusterId clusterId, chip::AttributeId attributeId,
 							  void *data, size_t dataSize)
 {
-	if (clusterId != Clusters::TemperatureMeasurement::Id || !data) {
+	CHIP_ERROR err = CHIP_NO_ERROR;
+	if (!data) {
 		return CHIP_ERROR_INVALID_ARGUMENT;
 	}
 
-	CHIP_ERROR err;
+	switch (clusterId) {
+	case Clusters::BridgedDeviceBasicInformation::Id:
+		err = HandleWriteDeviceBasicInformation(clusterId, attributeId, data, dataSize);
```

Comment:
nit
```suggestion
		return HandleWriteDeviceBasicInformation(clusterId, attributeId, data, dataSize);
```
And similarly below. You don't seem to need a single exit from the function so the function would be easier to understand (I wouldn't need to scroll down to the switch case end :)).